pkg/config/setup/config.go

```diff
@@ -1271,6 +1271,9 @@ func agent(config pkgconfigmodel.Setup) {
 
 	// Software Inventory
 	config.BindEnvAndSetDefault("software_inventory.enabled", false)
+	config.BindEnvAndSetDefault("software_inventory.jitter", 60)
+	config.BindEnvAndSetDefault("software_inventory.interval", 10)
+	bindEnvAndSetLogsConfigKeys(config, "software_inventory.forwarder.")
```

Comment:
This means customers will be able to configure API keys for software_inventory using `additional_endpoints`. You need to make sure API key refresh works as expected and please add a e2e test for it.